eea/api/dataconnector/api/dataconnector.py

```diff
@@ -41,12 +43,45 @@ def __call__(self, expand=False):
         return result
 
 
+@implementer(IExpandableElement)
+@adapter(IElasticDataProvider, Interface)
+class ElasticConnectorData(object):
+    """ Elastic connector data """
+
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+
+    def __call__(self, expand=False):
+        result = {
+            "connector-data": {
+                "@id": "{}/@connector-data".format(
+                    self.context.absolute_url()
+                )
+            }
+        }
+
+        if not expand:
+            return result
+
+        result["connector-data"][
+            "data"] = self.context.elastic_csv_widget.tableData
```

Comment:
This is dubious. If I have an expander called connector-data, I expect it would behave generic, it didn't have a specific implementation (ES)/.